### PR TITLE
Update release notes draft to version v1.29.0-alpha.2

### PR DESCRIPTION
**Pull request by fykaa**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.29.0-alpha.2

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Comment by rashansmith** [comment]
@fsmunoz @fykaa did we finalize whether all text should be encapsulated in '' vs. none at all? Otherwise everything looks good!

**Comment by fsmunoz** [comment]
> @fsmunoz @fykaa did we finalize whether all text should be encapsulated in '' vs. none at all? Otherwise everything looks good!

If there's the need for escaping things (and starting the line with backticks is one common reason, but there are more), both should work. If things are single-quoted, we shouldn't change them, if there's a need to quote it, both can work.

File: releases/release-1.29/release-notes/maps/pr-119732-map.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pr: 119732
+releasenote:
+  text: Fixed an issue related to not draining all the pods in a namespace when an
+    empty selector, i.e., "{}," is specified in a Pod Disruption Budget (PDB).
```
Comment (fsmunoz) [inline releases/release-1.29/release-notes/maps/pr-119732-map.yaml:4]:
```suggestion
  text: 'Fixed an issue related to not draining all the pods in a namespace when an empty selector, i.e., "{}," is specified in a Pod Disruption Budget (PDB).'
```

File: releases/release-1.29/release-notes/maps/pr-119735-map.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+pr: 119735
+releasenote:
+  text: Volume attach or publish operation will not fail at kubelet if target path
+    directory already exists on the node.
```
Comment (fsmunoz) [inline releases/release-1.29/release-notes/maps/pr-119735-map.yaml:4]:
```suggestion
  text: Volume attach or publish operation will not fail at `kubelet` if target path
    directory already exists on the node.
```

File: releases/release-1.29/release-notes/maps/pr-120338-map.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+pr: 120338
+releasenote:
+  text: 'k8s.io/dynamic-resource-allocation/controller: `UnsuitableNodes` can now
+    handle a mix of allocated and unallocated claims correctly.'
```
Comment (fsmunoz) [inline releases/release-1.29/release-notes/maps/pr-120338-map.yaml:4]:
```suggestion
  text: '`k8s.io/dynamic-resource-allocation/controller:` `UnsuitableNodes` can now
    handle a mix of allocated and unallocated claims correctly.'
```

File: releases/release-1.29/release-notes/maps/pr-120400-map.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+pr: 120400
+releasenote:
+  text: 'cluster-bootstrap: improved the security of the functions responsible for
+    generation and validation of bootstrap tokens'
```
Comment (fsmunoz) [inline releases/release-1.29/release-notes/maps/pr-120400-map.yaml:4]:
```suggestion
  text: '`cluster-bootstrap`: improved the security of the functions responsible for
    generation and validation of bootstrap tokens.'
```

File: releases/release-1.29/release-notes/maps/pr-120729-map.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+pr: 120729
+releasenote:
+  text: 'client-go: `k8s.io/client-go/tools` events and record packages now have new
+    APIs for specifying a context and logger.'
```
Comment (fsmunoz) [inline releases/release-1.29/release-notes/maps/pr-120729-map.yaml:4]:
```suggestion
  text: '`client-go`: `k8s.io/client-go/tools` events and record packages now have new
    APIs for specifying a context and logger.'
```

File: releases/release-1.29/release-notes/maps/pr-120828-map.yaml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+pr: 120828
+releasenote:
+  text: 'kubeadm: Fixed the bug that `kubeadm` always did CRI detection when `--config`
+    is passed even if it is not required by the subcommand.'
```
Comment (fsmunoz) [inline releases/release-1.29/release-notes/maps/pr-120828-map.yaml:4]:
```suggestion
  text: '`kubeadm`: Fixed the bug where it always did CRI detection when `--config`
    was passed, even if it is not required by the subcommand.'
```

File: releases/release-1.29/release-notes/maps/pr-120871-map.yaml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+pr: 120871
+releasenote:
+  text: |-
+    k8s.io/dynamic-resource-allocation: can now handle a `selected` node which isn't listed
+    as `potential` node.
```
Comment (fsmunoz) [inline releases/release-1.29/release-notes/maps/pr-120871-map.yaml:5]:
```suggestion
  text: |-
    `k8s.io/dynamic-resource-allocation`: can now handle a `selected` node which isn't listed
    as `potential` node.
```

File: releases/release-1.29/release-notes/maps/pr-120877-map.yaml [hunk 8]
```diff
@@ -0,0 +1,4 @@
+pr: 120877
+releasenote:
+  text: 'Fixed a regression (CLIENTSET_PKG: unbound variable) when invoking deprecated
+    `generate-groups.sh` script.'
```
Comment (fsmunoz) [inline releases/release-1.29/release-notes/maps/pr-120877-map.yaml:4]:
```suggestion
  text: 'Fixed a regression (`CLIENTSET_PKG: unbound variable`) when invoking deprecated
    `generate-groups.sh` script.'
```

**Review by fsmunoz — CHANGES_REQUESTED** [review]
Great work @fykaa ; these are some minor suggested changes.

**Comment by fykaa** [comment]
Thanks a lot for the review @fsmunoz @rashansmith. Resolved

**Comment by fsmunoz** [comment]
Great work @fykaa,  thanks @rashansmith for your review!

/lgtm 

Needs approval @kubernetes/release-team-leads

**Comment by Priyankasaggu11929** [comment]
/approve

Thanks everyone for the great work! :pray:

**Review by kumarankit999 — COMMENTED** [review]
Looks nice

**Pull request merged**